framework/tlvf/yaml/tlvf/ieee_1905_1/tlvLinkMetricQuery.yaml

```diff
@@ -2,29 +2,30 @@
 ---
 _namespace: ieee1905_1
 
-tlvLinkMetricQuery:
+# TLVF does not currently support optional fields and this TLV contains one.
+# As a workaround, we define two different TLVs instead of a single tlvLinkMetricQuery with an
+# optional field. Application must then check the length of received message to know if optional 
+# field is present or not and then create an instance of either of these classes.
+
+tlvLinkMetricQueryAllNeighbors:
```

Comment:
Since the tlvLinkMetricQuery class is removed here, we also have to remove all instances of it in the same commit. It is used in the parse function in CmduMessageRx and in tlvf_test.

To split up the commits a bit more, you could first simply add the two new ones but keep the old one around, then update the parse function and tlvf_test, and then remove the old one.

Reply:
With previous renaming, tlvLinkMetricQuery is being used back again.